public/usage-examples/graphics/current_clip/current-clip-1-simple.cpp

```diff
@@ -0,0 +1,24 @@
+#include "splashkit.h"
+
+int main() 
+{
+    // Open a window with the title "Current Clip" and dimensions
+    window win = open_window("Current Clip", 800, 600);
+
+    // Define the clipping area
```

Comment:
Make sure the commits are the same as well, so it makes less confusion to the users.